requirements.txt

```diff
@@ -2,7 +2,7 @@
 # apt install python3-dev default-libmysqlclient-dev build-essential redis-server supervisor
 
 # required packages  to use 'pip install -r requirements.txt'
-python
+#python

```

Comment:
Because python is needed at all ?

Reply:
apt install python3-dev is installing python. requirements is a pip import file, you cant import python from within python.